float64 time
float64 deltaTime
int16 right_forward
int16 right_side
int16 left_side
int16 left_forward